libs/core/langchain_core/language_models/llms.py

```diff
@@ -740,11 +756,11 @@ def generate(
             missing_prompt_idxs,
             missing_prompts,
         ) = get_prompts(params, prompts)
-        if isinstance(self.cache, BaseCache):
-            raise NotImplementedError(
-                "Local cache is not yet supported for " "LLMs (only chat models)"
-            )
-        disregard_cache = self.cache is not None and not self.cache
+        disregard_cache = (
```

Comment:
`disregard_cache = self.cache is not False`

All other cases including `self.cache is None` involve using a cache if it's configured